circle-pypy-base/Dockerfile

```diff
@@ -12,18 +12,18 @@ RUN apt-get -yq update && apt-get -yq install \
     liblapack-dev \
     gfortran
 
-ENV extra_index_url 'https://antocuni.github.io/pypy-wheels/ubuntu'
+ENV extra_index_url 'https://pypi.fury.io/alkaline-ml'
 
 # Make PyPy3 the default python rather than 2.7
 RUN ln -sf `which pypy3` /usr/bin/python
 
 # we have to run these in separate steps since the last thing pip does is
 # install at each step... first it collects. and subsequent packages require
 # installations (not just downloads) of previous packages in some cases.
-RUN python -m pip install --extra-index ${extra_index_url} numpy Cython pytest
-RUN python -m pip install --extra-index ${extra_index_url} scipy
-RUN python -m pip install --extra-index ${extra_index_url} scikit-learn
-RUN python -m pip install --extra-index ${extra_index_url} "pandas==0.23.*" statsmodels matplotlib
-RUN python -m pip install --extra-index ${extra_index_url} pytest-mpl pytest-benchmark
+RUN python -m pip install --extra-index ${extra_index_url} --prefer-binary numpy Cython pytest
```

Comment:
That's cool, I didn't know `prefer-binary` was a thing.

Reply:
I've learned way more about `pip` than I would like with all of this 😄